tests/testthat/comparison_data/gcam-usa/L143.HDDCDD_scen_state.csv

```diff
@@ -1,6 +1,6 @@
 # ../gcam-usa-data/level1//L143.HDDCDD_scen_state.csv
 # Written by LA143.HDDCDD.R
-# Wed Sep 20 02:00:41 2017
+# Mon Nov 20 14:29:22 2017
 # " Heating and cooling degree days by state and scenario "
```

Comment:
We must track down the source of the HDD and CDD data. I am not sure off the top of my head. @pkyle , any thoughts?

Reply:
I see that the original files are in the GIS folder within gcam-usa-data. Is that where we plan to keep these files?

Reply:
@gokuliyer @bpbond @pkyle I put the CDD and HDD input files into the `gcamdata\inst\extdata\gcam-usa` but @gokuliyer is right, in the old data system it was in the `gcam-usa-data\GIS` directory. Where do we want to put it in the new data system.

Reply:
I'd suggest `gcamdata\inst\extdata\gcam-usa\GIS\`

Reply:
Sorry guys, I just noticed I hadn't written back!
The HDD and CDD data in the model were all produced by Yuyu, and we don't have any archive of the raw data, processing code, etc. You can see from the file names that the future scenarios are from the GFDL climate model, A2 scenario; the historical ones are also from a climate model as well (as opposed to historical or re-analysis data), but the fact that they only go up to 2008 means that the dataset was produced in 2011. And I don't know which climate model was used--my guess is CCSM3. Here's an excerpt from some documentation I wrote up in 2012:
"Historical gridded population data for the US are from HYDE (Klein Goldewikj et al. 2011), which has population at the 5’ scale. Global gridded population data for future scenarios are from the IIASA GGI database. These scenarios are available at 0.5 degree latitude by longitude at a 10-year resolution from 2000 to 2100. Historical and future climate data are from three climate models, depending on the scenario being run. These models include CCSM3 (Collins et al. 2006), GISS (Russell et al. 2000), and Hadley (Gordon et al. 2000). These population and climate data are used to generate historical heating and cooling degree days (HDD and CDD), as well as future scenarios."...
"The method for calculating heating and cooling degree days by state using gridded data of temperature and population over time is documented in Zhou et al. (in review); a brief summary follows. First, the monthly temperature data and estimates of the daily standard deviation in each grid cell, derived from analysis of historical climate data, are used to calculate the monthly HDD and CDD for each grid cell. This step is necessary because climate model output for future scenarios is in monthly timesteps, whereas heating and cooling degree days are calculated from daily temperature. Then, these monthly HDD and CDD are aggregated to annual values.Then, the population-weighted HDD and CDD over a given geographical area (in this case, states) are calculated as follows:"
![hddcddcalc](https://user-images.githubusercontent.com/20807787/33686112-c7da6394-daa1-11e7-9c7f-bb35a4521226.png)

Reply:
Thanks @pkyle!